juriscraper/opinions/united_states/state/nd.py

```diff
@@ -50,11 +51,23 @@ def _process_html(self) -> None:
 
             for idx, txt in enumerate(raw_values[:5]):
                 if idx == 0:
-                    txt, _ = self.clean_name(txt)
+                    # Separate case name and citation if present
```

Comment:
looks like you dropped clean name completely ... is this not needed anymore?  how did we get passed it 

and it looks like if we dont match on a citation ... like the example below this will crash because we no citation